editor/src/components/canvas/canvas-strategies/strategies/keyboard-set-font-size-strategy.tsx

```diff
@@ -0,0 +1,119 @@
+import {
+  getSimpleAttributeAtPath,
+  MetadataUtils,
+} from '../../../../core/model/element-metadata-utils'
+import { elementOnlyHasTextChildren } from '../../../../core/model/element-template-utils'
+import { mapDropNulls } from '../../../../core/shared/array-utils'
+import { defaultEither, isLeft, right } from '../../../../core/shared/either'
+import { ElementInstanceMetadataMap, isJSXElement } from '../../../../core/shared/element-template'
+import { ElementPath } from '../../../../core/shared/project-file-types'
+import * as PP from '../../../../core/shared/property-path'
+import {
+  cssNumber,
+  CSSNumber,
+  parseCSSLengthPercent,
+  printCSSNumber,
+} from '../../../inspector/common/css-utils'
+import { setProperty } from '../../commands/set-property-command'
+import {
+  InteractionCanvasState,
+  CanvasStrategy,
+  getTargetPathsFromInteractionTarget,
+  emptyStrategyApplicationResult,
+  strategyApplicationResult,
+} from '../canvas-strategy-types'
+import { InteractionSession } from '../interaction-state'
+import { accumulatePresses } from './shared-keyboard-strategy-helpers'
+
+export function keyboardSetFontSizeStrategy(
+  canvasState: InteractionCanvasState,
+  interactionSession: InteractionSession | null,
+): CanvasStrategy | null {
+  const validTargets = getTargetPathsFromInteractionTarget(canvasState.interactionTarget).filter(
+    (path) => {
+      const element = MetadataUtils.findElementByElementPath(canvasState.startingMetadata, path)
+      if (element == null || isLeft(element.element) || !isJSXElement(element.element.value)) {
+        return false
+      }
+      return elementOnlyHasTextChildren(element.element.value)
+    },
+  )
+
+  if (validTargets.length === 0) {
+    return null
+  }
+
+  return {
```

Comment:
We'll either need to return `null` if there is an existing mouse interaction session, or return a fitness of `0` in that case. In other cases we've returned a fitness of `0` unless there is a keyboard interaction session with the valid keys pressed, e.g. <https://github.com/concrete-utopia/utopia/blob/f37ca681d77099e63c6b652bbe8658b89721a9e1/editor/src/components/canvas/canvas-strategies/strategies/keyboard-absolute-move-strategy.ts#L121-L135>

Reply:
fixed in [694e8dc](https://github.com/concrete-utopia/utopia/pull/2993/commits/694e8dc2e15d492e9558d493e0c45138dc189265)!